assets/less/app.less

```diff
@@ -1,5 +1,5 @@
 @import 'npm://font-awesome/less/font-awesome';
-@import 'npm://highlightjs/styles/tomorrow';
+@import 'npm://highlight.js/src/styles/tomorrow';
```

Comment:
because now we are using a lot of projects. i think it will be hard to keep track of everything. what should we do?